packages/react-flex/Spec.md

```diff
@@ -0,0 +1,524 @@
+| FlexItem           | `size`          | Renamed to `basis`.                                    |
+
+## Sample Code
+
+### Basic
+
+```HTML
+<Flex>
+ <span>1</span>
+ <span>2</span>
+</Flex>
+```
+
+### Direction
+
+```HTML
+<Flex direction="column-reverse">
+  <span>bottom</span>
+  <span>top</span>
+</Flex>
+```
+
+### Alignment
+
+```HTML
+<Flex verticalAlign="center">
+  <span>vertically centered</span>
+  <span>vertically centered</span>
+</Flex>
+```
+
+```HTML
+<Flex horizontalAlign="center">
+  <span>horizontally centered</span>
+  <span>horizontally centered</span>
+</Flex>
+```
+
+### Spacing
+
+```HTML
+<Flex gap="5rem">
+  <span>5rem around me</span>
+  <span>5rem around me</span>
+</Flex>
+```
+
+### Wrapping
+
+```HTML
+<Flex wrap>
+  <span>1</span>
+  <span>2</span>
+</Flex>
+```
+
+### Order
+
+```HTML
+<Flex>
+  <span>2</span>
+  <span order={1}>1</span>
+</Flex>
+```
+
+### Growing
+
+#### Globally
+
+```HTML
+<Flex grow>
+  <span>wide</span>
+  <span>wide</span>
+</Flex>
+```
+
+#### Individually
+
+```HTML
+<Flex>
+  <span>thin</span>
+  <span grow>wide</span>
+</Flex>
+```
+
+### Shrinking
+
+#### Globally
+
+```HTML
+<Flex shrink={false}>
+  <span>wide</span>
+  <span>wide</span>
+</Flex>
+```
+
+#### Individually
+
+```HTML
+<Flex>
+  <span>thin</span>
+  <span shrink={false}>wide</span>
+</Flex>
+```
+
+### Complex example
+
+```HTML
+<Flex
+  as="section"
+  direction="row-reverse"
+  horizontalAlign="space-between"
+  verticalAlign="center"
+  gap="5rem"
+  wrap grow inline>
+  <span order={3}>3</span>
+  <span shrink={false}>1</span>
+  <span basis="25%">2</span>
+</Flex>
+```
+
+## Migration
+
+_Describe what will need to be done to upgrade from the existing implementations:_
+
+### _Migration from v8_
+
+#### Stack
+
+##### `disableShrink`
+
+Renamed from `disableShrink` to `shrink`
+
+Before
+
+```HTML
+<Stack disableShrink>
+ items
+</Stack>
+```
+
+After
+
+```HTML
+<Flex shrink="0">
+  items
+</Flex>
+```
+
+```HTML
+<Flex shrink={false}>
+  items
+</Flex>
+```
+
+##### `horizontal`
+
+Before
+
+```HTML
+<Stack horizontal>
+ items
+</Stack>
+```
+
+After
+
+```HTML
+<Flex>
+  items
+</Flex>
+```
+
+##### `childrenGap`
+
+Renamed from `childrenGap` to `gap`
+
+##### `reversed` [DEPRECATED]
+
+Before
+
+```HTML
+<Stack reversed>
+ items
+</Stack>
+```
+
+Alternative
+
+```HTML
+<Flex direction="column-reverse">
+  items
+</Flex>
+```
+
+##### `maxHeight` [DEPRECATED]
+
+Before
+
+```HTML
+<Stack maxHeight="30px">
+ items
+</Stack>
+```
+
+Alternative
+
+```HTML
+<Flex style={{maxHeight:"30px"}}>
+  items
+</Flex>
+```
+
+##### `maxWidth` [DEPRECATED]
+
+Before
+
+```HTML
+<Stack maxWidth="30px">
+ items
+</Stack>
+```
+
+Alternative
+
+```HTML
+<Flex style={{maxWidth:"30px"}}>
+  items
+</Flex>
+```
+
+##### `verticalFill` [DEPRECATED]
+
+Before
+
+```HTML
+<Stack verticalFill>
+ items
+</Stack>
+```
+
+Alternative
+
+```HTML
+<Flex style={{height: "100%"}}>
+  items
+</Flex>
+```
+
+##### `padding` [DEPRECATED]
+
```

Comment:
**q:** no example. on purpose/typo?

Reply:
Definitely a typo but it will be basically the same as the other `padding` prop deprecation. Will fix